media/visual.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="D:/Projects/Programming/callstackninja/media/visual.css">
```

Comment:
use reference path instead of absolute

Reply:
It is solved using vscode.Uri instead of a reference path.